doc/lua_api.txt

```diff
@@ -2271,18 +2271,20 @@ Examples
 Version History
 ---------------
 
-* FORMSPEC VERSION 1:
+* Formspec version 1 (old):
   * (too much)
-* FORMSPEC VERSION 2:
+* Formspec version 2 (5.1):
```

Comment:
```suggestion
* Formspec version 2 (5.1.0):
```
same for the others to clarify that it's a version, and not some random floating point number.